modules/ossm-config-openshift-monitoring.adoc

```diff
@@ -0,0 +1,136 @@
+:_mod-docs-content-type: Procedure
+[id="ossm-config-openshift-monitoring"]
+= Configuring OpenShift Monitoring with Service Mesh
```

Comment:
I think this can actually be broken up into two sections: 
1. Configuring Istio with workload monitoring 
2. Installing Kiali / configuring kiali with workload monitoring. 

or even more simply:
Observability:
- Metrics (here's where you have how to configure istio with workload monitoring)
- Kiali (here's where you have how to install kiali and configure _kiali_ with workload monitoring)

wdyt?

Reply:
Good to know. 

For the purposes of TP1, Kiali + Metrics is what we're documenting. That may change between TP 1 and GA, but for TP 1, just documenting Kiali + Metrics.